docs/user-guide/configuring.md

```diff
@@ -208,19 +208,19 @@ To specify globals using a comment inside of your JavaScript file, use the follo
 /* global var1, var2 */
 ```
 
-This defines two global variables, `var1` and `var2`. If you want to optionally specify that these global variables should never be written to (only read), then you can set each with a `false` flag:
+This defines two global variables, `var1` and `var2`. If you want to optionally specify that these global variables can be written to (rather than only being read), then you can set each with a `writeable` flag:
 
 ```js
-/* global var1:false, var2:false */
+/* global var1:writeable, var2:writeable */
```

Comment:
"writeable" and "writable" are both valid spellings. Although ["writable" is apparently more common](https://trends.google.com/trends/explore?geo=US&q=writable,writeable), I decided to use "writeable" for consistency with property from scope analysis exposed to rules, which is already called "writeable".

Reply:
We default to "writeable" for unknown values anyway, so technically "writable" is already supported as an alias (along with everything else). If we start reporting an error for unknown values in a major version I think it would be reasonable to keep tolerating "writable".